contents/handbook/growth/marketing/ownership.md

```diff
@@ -30,19 +30,19 @@ showTitle: true
 | **Changelog**                           | Joe                  |
 | **Emailing customers**                  | Joe                  |
 | **Events**                              | Joe                  |
+| **Growth review**                       | Brian                |
 | **Incident response**                   | Joe                  |
 | **In-app notifications**                | Joe                  |
-| **Merch design**                        | Lottie               |
-| **Merch promotion**                     | Lottie               |
-| **Merch stock / shipping**              | Kendall              |
 | **Onboarding emails**                   | Joe                  |
-| **Paid Ads**                            | Joe/Andy             |
+| **Paid ads strategy and messaging**     | Brian                |
+| **Paid ads design**                     | Hey                  |
+| **Paid ads operations**                 | Hey                  |
```

Comment:
```suggestion
| **Paid ads design**                     | Brian                  |
| **Paid ads operations**                 | Brian                  |
```
I think it is clearer if it sits with Brian. He manages the agency, so has responsibility for their actions. Hey are just a tool Brian uses to achieve things now and don't own anything directly - in the same way that GitHub doesn't have ownership of anything.